sdk/eventhub/event-hubs/samples-dev/sendBufferedEvents.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/**
+ * @summary Demonstrates how to send events to an Event Hub from a buffered producer. This
+ * sample is different from the `sendEvent` one in that it uses the buffered producer
+ * client which buffers events before sending them right away. See the client constructor
+ * options for more information on how to configure the buffering behavior.
```

Comment:
How about this?

```suggestion
 * @summary Demonstrates how to send events to an Event Hub using the `EventHubBufferedProducerClient`. 
 * This sample is different from the one in `sendEvent.ts` in that the client manages batching of events and sending
 * after a given amount of time or after a given amount of events are in a batch instead of you managing the same explicitly.
```

Reply:
done!